tox.ini

```diff
@@ -30,8 +30,6 @@ deps =
     py37: numpy==1.14.5
     py37: scipy==1.1.0
 
-    py27: numpy==1.11.3
-    py27: scipy==0.18.1
```

Comment:
need also to remove ` --trusted-host 28daf2247a33e...` I guess too

Reply:
What is that stuff? I'm not particularly familiar with them

Reply:
Let's discuss that on telegram